docs/mobile_data_collected.md

```diff
@@ -0,0 +1,173 @@
+
+| `resource.provider.name`      | string | The resource provider name. Default is `unknown`.                                            |
+| `resource.provider.domain`      | string | The resource provider domain.                                            |
+| `resource.provider.type`      | string | The resource provider type (for example `first-party`, `cdn`, `ad`, `analytics`).                                            |
+
+
+
+{{% /tab %}}
+{{% tab "Error" %}}
+
+Front-end errors are collected with Real User Monitoring (RUM). The error message and stack trace are included when available.
+
+### Error attributes
+
+| Attribute       | Type   | Description                                                       |
+|-----------------|--------|-------------------------------------------------------------------|
+| `error.source`  | string | Where the error originates from (for example, `webview`, `logger` or `network`).     |
+| `error.type`    | string | The error type (or error code in some cases).                   |
+| `error.message` | string | A concise, human-readable, one-line message explaining the event. |
+| `error.stack`   | string | The stack trace or complementary information about the error.     |
+
+
+#### Network errors
+
+Network errors include information about failing HTTP requests. The following facets are also collected:
+
+| Attribute                      | Type   | Description                                                                             |
+|--------------------------------|--------|-----------------------------------------------------------------------------------------|
+| `error.resource.status_code`             | number | The response status code.                                                               |
+| `error.resource.method`                | string | The HTTP method (for example `POST`, `GET`).           |
+| `error.resource.url`                     | string | The resource URL.                                                                       |
+| `error.resource.provider.name`      | string | The resource provider name. Default is `unknown`.                                            |
+| `error.resource.provider.domain`      | string | The resource provider domain.                                            |
+| `error.resource.provider.type`      | string | The resource provider type (for example `first-party`, `cdn`, `ad`, `analytics`).                                            |
+
+
+{{% /tab %}}
+{{% tab "User Action" %}}
+
+
+
+### Action metrics
+
+| Metric    | Type   | Description              |
+|--------------|--------|--------------------------|
+| `action.loading_time` | number (ns) | The loading time of the action.  |
+| `action.resource.count`         | number      | Count of all resources collected for this action. |
+| `action.error.count`      | number      | Count of all errors collected for this action.|
```

Comment:
"collected by" sounds technical. The user observation is IMO better expressed by "issued by" (as the action may result with resource start or error occurrence):
```suggestion
| `action.resource.count`         | number      | Count of all resources issued by this action. |
| `action.error.count`      | number      | Count of all errors issued by this action.|
```